BUILD

```diff
@@ -15,7 +15,10 @@
 java_library(
     name = "common-core",
     srcs = glob(["core/src/main/java/com/google/instrumentation/common/*.java"]),
-    deps = ["@jsr305//jar"],
+    deps = [
+        "@guava//jar",
```

Comment:
We discussed this with gRPC people. As long as we depend on the same guava version is good. We use guava in trace a lot for checkNotNull, checkArgument, etc.